internal/service/ivs/status.go

```diff
@@ -10,6 +10,9 @@ import (
 
 const (
 	statusNormal = "Normal"
+
+	statusActive   = "ACTIVE"
+	statusCreating = "CREATING"
```

Comment:
Use `ivs.RecordingConfigurationStateActive` and `ivs.RecordingConfigurationStateCreating` from the AWS Go SDK.